src/App.tsx

```diff
@@ -1,24 +1,133 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  ChangeEvent, useCallback, useEffect, useState,
+} from 'react';
+
+import { getTodos, addTodo, removeTodo } from './api/todos';
 import { UserWarning } from './UserWarning';
+import { Todo, ErrorTypes } from './types/types';
+import { Filter } from './components/Filter/Filter';
+import { NewTodo } from './components/NewTodo/NewTodo';
+// eslint-disable-next-line max-len
+import NotificationError from './components/NotificationError/NotificationError';
 
-const USER_ID = 0;
+const USER_ID = 6982;
 
 export const App: React.FC = () => {
+  const [task, setTask] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState(ErrorTypes.none);
+  const [tempTodo, setTemptodo] = useState<Todo | null>(null);
+  const [isDataUpdated, setIsdataUpdated] = useState(false);
+  const [activeIds, setActiveIds] = useState([0]);
+
+  const handleTodoChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setTask(event.target.value);
+  };
+
+  const resetTempTodo = useCallback(() => {
+    setTemptodo(null);
+    setActiveIds([0]);
+  }, []);
+
+  const loadTodos = async () => {
+    setIsdataUpdated(true);
+
+    try {
+      const todosFromServer = await getTodos(Number(USER_ID));
+
+      setTodos(todosFromServer);
+    } catch {
+      setError(ErrorTypes.get);
+    } finally {
+      setIsdataUpdated(false);
+      resetTempTodo();
+    }
+  };
+
+  const handleTodoSubmit = async (event: ChangeEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (!task.trim().length) {
+      setError(ErrorTypes.emptyTitle);
+      setTask('');
+
+      return;
+    }
+
+    setTemptodo({
+      id: 0,
+      userId: USER_ID,
+      title: task,
+      completed: false,
+    });
+    setTask('');
+
+    try {
+      await addTodo({
```

Comment:
You should use the 'tempTodo' object instead of creating a new one when calling 'addTodo'. This ensures the added todo is consistent with 'tempTodo'.